strings/isisogram.go

```diff
@@ -0,0 +1,54 @@
+// Checks if a given string is an isogram.
+// A first-order isogram is a word in which no letter of the alphabet occurs more than once.
+// A second-order isogram is a word in which each letter appears twice.
+// A third-order isogram is a word in which each letter appears three times.
+// wiki: https://en.wikipedia.org/wiki/Heterogram_(literature)#Isograms
+// Author: M3talM0nk3y
+
+package strings
+
+import (
+	"regexp"
+	"strings"
+)
+
+func hasDigit(text string) bool {
+	re := regexp.MustCompile(`\d`)
+	return re.MatchString(text)
+}
+
+func hasSymbol(text string) bool {
+	re := regexp.MustCompile(`[-!@#$%^&*()+]`)
+	return re.MatchString(text)
+}
+
+func IsIsogram(text string) bool {
```

Comment:
When I suggested the previous change, I didn't mean that you should modify your function such that the user doesn't know which Isogram it is. Currently, the caller of this function has no idea whether they received first, second or third order isogram. My point was to change this function such that it takes the order as argument.

```suggestion
func IsIsogram(s string, order int) (bool, error) {
	if order == 0 {
		return false, errors.New("please provide the order to check for ordered isogram the string")
	}
```

Reply:
Thanks for the feedback, Taj. Made the suggested changes.